docs/en/installation.rst

```diff
@@ -1,16 +1,18 @@
-Installation
+Installation of IDF
+===============================
+Installing the ESP-IDF using EIM is straightforward. Simply open your preferred shell (we recommend PowerShell for Windows users) and run EIM, providing your installation preferences using any of the configuration <configuration> methods you choose. If you haven't provided all the necessary options, an interactive wizard will guide you through the remaining steps.
+
+After Installation 
```

Comment:
Please describe the installation process of the eim tool in detail, including screenshots. 

Also, explain what artifacts it will generate, where they will be stored, and how users should get started once the installation is complete.